### ISSUE TITLE
Canonicalise blue-shirt vs blueshirt vs Blueshirt

### ISSUE THREAD

**Issue by PeterJCLaw** [srobo/tasks#1471]
We have a couple of spellings for this friendly name for our volunteers, we should pick one and stick to it.

At least the docs and website will need updating once we've decided, since they each currently have examples of both spellings.

**Comment by samjmartin04** [comment]
I feel as long as I've been involved, the preference seems to have been toward Blueshirt, with blueshirt sometimes being used. I've very rarely seen blue-shirt.

**Comment by PeterJCLaw** [comment]
Yeah, agree we usually have it as one word. I think that's what's in our glossary too.

**Comment by samjmartin04** [comment]
I'm fairly sure Blueshirt capitalised is much more common than blueshirt so I'm not sure this should necessarily have been closed with 'blueshirt' as the decision.

**Comment by PeterJCLaw** [comment]
Oh, sorry, forgot that was a consideration.

**Comment by samjmartin04** [comment]
No worries. Grammar wise I think capitalised makes sense when referring to a group of people?

**Comment by PeterJCLaw** [comment]
I think we need to decide if it's a proper noun or not. I don't think the group/non-group or people aspects matter?

**Comment by PeterJCLaw** [comment]
IMO this isn't a proper noun -- compare to "police officer" or "teacher", which I think are comparable nouns. That would mean "blueshirt" over "Blueshirt" (unless other grammatical rules indicated otherwise, of course).

**Comment by PeterJCLaw** [comment]
The counterpoint here might be that many people may not know what a "blueshirt" is nor why it'd different to a "blue shirt", so treating this as a proper noun could help clarify that?

**Comment by PeterJCLaw** [comment]
However in that regard we should probably work out what we want to do about "Kickstart", "Competition" and other terms to which we attach specific meaning. Do we have a precedent around those?

**Comment by samjmartin04** [comment]
I think treating Blueshirt as a proper noun would make it clearer. Blueshirt with a B makes it clearer that it is a group of people and not just a blue shirt (or typo of that). 

We tend to use Kickstart and Competition with capitalised letters

**Comment by sedders123** [comment]
Discussion between Marketing Committee (+ Event Logistics its a rambly meeting) has ended with Blueshirt should have a capital B

**Comment by sedders123** [comment]
Woops closed before PR-ing
